docker/kites/net-tests-dev/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM debian:10
+RUN apt-get update \
+    && apt-get install -y iperf3 \
```

Comment:
1) move iperf3 to a new line
2) sort the package list alphabetically
3) (hadolint) DL3008 Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`
4) (hadolint) DL3015 Avoid additional packages by specifying `--no-install-recommends`